WPFHospitalEditorUnitTests/SearchAppointmentTest.cs

```diff
@@ -0,0 +1,289 @@
+﻿using HealthcareBase.Model.HospitalResources;
+using HealthcareBase.Model.Schedule.Procedures;
+using HealthcareBase.Model.Schedule.SchedulingPreferences;
+using HealthcareBase.Model.Users.Employee.Doctors;
+using HealthcareBase.Model.Users.Generalities;
+using HealthcareBase.Model.Utilities;
+using HealthcareBase.Repository.ScheduleRepository.ProceduresRepository.Interface;
+using HealthcareBase.Repository.UsersRepository.EmployeesAndPatientsRepository.Interface;
+using HealthcareBase.Service.ScheduleService.ProcedureService;
+using HospitalWebApp.Dtos;
+using Moq;
+using Shouldly;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Windows;
+using Xunit;
+
+namespace WPFHospitalEditorUnitTests
+{
+    public class SearchAppointmentTest
+    {
+        private readonly Mock<IShiftRepository> mockShiftRepository = new Mock<IShiftRepository>();
+        private readonly Mock<IExaminationRepository> mockExaminationRepository = new Mock<IExaminationRepository>();
+        private readonly Mock<IDoctorRepository> mockDoctorRepository = new Mock<IDoctorRepository>();
+        private readonly ExaminationService examinationService;
+        private Doctor doctor1;
+        private Doctor doctor2;
+        private Doctor doctor3;
+        private TimeInterval timeInterval1;
+        private TimeInterval timeInterval2;
+        private TimeInterval timeInterval3;
+        private Room room1;
+        private Room room2;
+        private Shift shift1;
+        private Shift shift2;
+        private Shift shift5;
+        private RecommendationRequestDto dto1;
+        private RecommendationRequestDto dto2;
+        private RecommendationRequestDto dto3;
+        private RecommendationRequestDto dto4;
+        private RecommendationRequestDto dto5;
+        private RecommendationRequestDto dto6;
+        private List<Shift> shifts1 = new List<Shift>();
+        private List<Shift> shifts2 = new List<Shift>();
+        private List<Shift> shifts3 = new List<Shift>();
+
+        public SearchAppointmentTest()
+        {
+            
+            examinationService = new ExaminationService(mockExaminationRepository.Object, mockShiftRepository.Object, mockDoctorRepository.Object);
+
+
+            InitDoctors();
+            InitTimeInterval();
+            InitRooms();
+            InitDoctorShifts();
+            InitRecommendationDto();
+        }
+
+        [Fact]
+        public void Find_5_appointments_time_priority()
+        {
+
+            StubRepositories();
+            List<RecommendationDto> availableAppointments = new List<RecommendationDto>();
+
```

Comment:
Do you use availableAppointments in any test?

Reply:
No, I deleted it.